Lib/glyphsLib/builder/user_data.py

```diff
@@ -92,6 +92,8 @@ def to_ufo_layer_user_data(self, ufo_glyph, layer):
 
 def to_ufo_node_user_data(self, ufo_glyph, node):
     user_data = node.userData
+    if "name" in user_data:  # Handle point name at a higher level.
+        del user_data["name"]
```

Comment:
Why does this function modify the Glyphs object in "to_ufo"? It should only read from the Glyphs object